.github/paths-labeller.yml

```
@@ -0,0 +1,3 @@
+---
+  - "Team: Ingest Management":
+    - "x-pack/elastic-agent/**/*.*"
```

NIT: can we add the below one for the automation team?

```suggestion
  - "Team: Ingest Management":
    - "x-pack/elastic-agent/**/*.*"
  - "Team: Automation":
    - ".ci/**/*.*"
    - "Jenkinsfile"
```